perception/tensorrt_yolox/package.xml

```diff
@@ -23,6 +23,7 @@
   <depend>cuda_utils</depend>
   <depend>cv_bridge</depend>
   <depend>image_transport</depend>
+  <depend>image_transport_decompressor</depend>
```

Comment:
I guess this is execution dependency.

Reply:
changed at [4536ca8](https://github.com/autowarefoundation/autoware.universe/pull/2982/commits/4536ca87c2835d12efbdc43671087b6847c211c5)